### ISSUE TITLE
Add and serve swagger api docs, i.e. spec.yaml? 

### ISSUE THREAD

**Issue by mvolz** [zotero/translation-server#76]
If I made a pull request for adding and serving swagger docs, would that something you would merge? 
https://swagger.io/specification/ 

Example of what swagger docs look like in the wild: https://en.wikipedia.org/api/rest_v1/#!/Citation/getCitation

**Comment by dstillman** [comment]
Sure!

**Comment by retorquere** [comment]
Has work on this been done? I'd be very interested in a swagger spec for the zotero api!

**Comment by mvolz** [comment]
> Has work on this been done? I'd be very interested in a swagger spec for the zotero api!

Nope- WIP here, but never got around to submitting this upstream T_T https://gerrit.wikimedia.org/r/#/c/mediawiki/services/zotero/+/481170/
